galaxy_skeleton2.R

```diff
@@ -91,16 +99,42 @@ comma_separated_values_with_names <-
   c(paste(variable_names, collapse = ","),
     comma_separated_values)    
 
+comma_separated_values_with_names <- 
+  c(paste(variable_names, collapse = ","),
+    comma_separated_values)
+
+
+data_file_name <- "galaxy_data.csv"
 # Replace the question mark and come up with a file name
-cat(?, sep = "\n", file = "?")
 
+
+
+cat(comma_separated_values_with_names, sep = "\n", file = data_file_name)
 # Read the file back in as a normal csv-file. The readr-package is part of
 # tidyverse, so it is already loaded.
-galaxies <- read_csv("?")
+galaxies <- read_csv(data_file_name)
 
 
 # You should now have a nice, clean data frame with galaxies and their
 # characteristics in memory. As of March 2022 it should contain 796
 # observations.
+galaxies <- galaxies[-1, ]
+galaxies
+
+
+
+##### Problem 3 #######
+library(ggplot2)
+
+# Create a histogram of the linear diameter (a_26) variable
+ggplot(galaxies, aes(x = a_26)) +
+  geom_histogram(binwidth = 1, fill = "blue", color = "black") +
+  labs(
+    title = "Distribution of Galaxy Linear Diameters",
+    x = "Linear Diameter (kpc)",
+    y = "Count"
+  ) +
+  theme_minimal()
 
 
+#doesn't it mean that, smaller galaxies are over represented?
```

Comment:
Not exactly. You can try to plot a scatter plot of size vs distance. In this plot it seems like there are more galaxies close to us. This might indicate that our technology struggle in noticing small galaxies far away.